### PR TITLE
merge feature/dockerize to develop

### PR DESCRIPTION
**Pull request by visualizeq**
Stable version for current build of container

Changes:
- added TimeZone (`TZ`) in container (Asia/Bangkok)
- add docker build script for easy build container image
- add `JOBS=$(nproc)` for `node-medley` to speed up native build time
- add missing devDependencies in `radio` (`@types/dotenv`)

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# TODO: Move this file into packages/radio
```
Comment (vittee) [inline docker/Dockerfile:1]: Retain this

File: docker/Dockerfile [hunk 2]
```diff
@@ -14,11 +15,33 @@ RUN git submodule update --init --force --recursive
...
+WORKDIR /app
 # cleanup unnecessary files/folders
 RUN rm -rfv .git .github .dockerignore .editorconfig .gitignore
 RUN rm -rfv /app/packages/radio/.env
 
+RUN npm i pnpm -g
+
 WORKDIR /app/packages/radio
 
-# TODO: Remove this and use command for starting radio bot
```
Comment (vittee) [inline docker/Dockerfile:23]: Retain this

Comment (visualizeq) [reply docker/Dockerfile:23]: already merge [here](https://github.com/vittee/medley/pull/5/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR45-R46) 😂

File: docker/Dockerfile (hunk 1 above)
Comment (visualizeq) [reply docker/Dockerfile:1]: done here b9d17a524cc8c4576c43a2b9fb8d787412d13e62

**Pull request merged**